src/Swarrot/Processor/Retry/RetryProcessor.php

```diff
@@ -61,7 +61,11 @@ public function process(Message $message, array $options)
                 $properties
             );
 
-            $key = str_replace('%attempt%', $attempts, $options['retry_key_pattern']);
+            $key = str_replace(
+                '%attempt%',
+                ($attempts > $options['retry_attempts']) ? $options['retry_attempts'] : $attempts,
```

Comment:
If I understand well, once the retry_attempts is reached, it will republish in the routing_key as before?
Not sure it's very understandable if the processor continue to publish in `something_retry_3` (for example) after 10 attempts. :/
In your use case is it possible to have a very high attempts limit and a routing_key which doesn't contains %retry%?